apps/design-system/app/Providers.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import * as React from 'react'
+import { Provider as JotaiProvider } from 'jotai'
```

Comment:
curious why not valtio?

Reply:
Not sure it matters.
Large bits of the site was from ui.shadcn and they used jotai.

[This tweet](https://x.com/dai_shi/status/1343694177884721152) sums up the differences 😂

Reply:
aah got it — kk just curious